scripts/caniuse.ts

```diff
@@ -20,6 +20,8 @@ const mapping = new Map<string, string | null>(
     Object.keys(lite.features).sort().map(id => [id, null])
 );
 
+const hiddenCaniuseItems = Object.entries(lite.features).flatMap(([id, data]) => !lite.feature(data).shown ? [id] : []);
```

Comment:
```suggestion
const hiddenCaniuseItems = new Set(Object.entries(lite.features).flatMap(([id, data]) => !lite.feature(data).shown ? [id] : []));
```

Reply:
But also, this one-liner takes a bit of work to decode. I would understand this faster:

```js
const hiddenCaniuseItems = new Set();
for (const [id, data] of Object.entries(lite.features)) {
  if (!lite.feature(data).shown) {
    hiddenCaniuseItems.add(id);
  }
}
```

Reply:
I always forget `Set` is a thing. I much prefer this approach (and the unrolling of the flatmap, which was a bit too clever to begin with).